src/SqlClient/SqlClient.fsproj

```diff
@@ -3,17 +3,11 @@
   <Import Project="../../netfx.props" />
   <Import Project="../../fsc.props" />
   <PropertyGroup>
-    <TargetFramework>netstandard20</TargetFramework>
+    <TargetFrameworks>net462;netstandard20</TargetFrameworks>
```

Comment:
moved net462 to the front because ionide uses the first one to provide completions/errors